volume-providers/vpc/vpcclient/models/volume_attachment.go

```diff
@@ -77,5 +84,20 @@ func (va *VolumeAttachment) ToVolumeAttachmentResponse() *provider.VolumeAttachm
 	if va.InstanceID != nil {
 		varp.InstanceID = *va.InstanceID
 	}
+
+	//Set DevicePath
+	if va.Device != nil && va.Device.ID != "" {
+		devicepath := va.Device.ID
+		generation := "gc" //default
+		if va.Volume != nil && va.Volume.Generation != "" {
+			generation = va.Volume.Generation.String()
+		}
+
+		//prepend "/dev/" for generation=1 (gc)
+		if generation == "gc" && !strings.HasPrefix(devicepath, "/dev/") {
+			devicepath = "/dev/" + va.Device.ID
```

Comment:
Lets keep const variable for all inline hardcoded values i.e `/dev/`

Reply:
done